migrations/migrations.sql

```diff
@@ -4303,5 +4303,18 @@ do $$
     if not exists (select 1 from information_schema.columns where table_name = 'accountIntegrations' and column_name = 'isInternal') then
       alter table "accountIntegrations" add column "isInternal" BOOLEAN;
     end if;
+
+    -- Add projectId column to jobTestReports table and populate projectId for existing jobTestReports
+    if not exists (select 1 from information_schema.columns where table_name = 'jobTestReports' and column_name = 'projectId') then
+      alter table "jobTestReports" add column "projectId" varchar(24);
+      update "jobTestReports" as t set "projectId" = (select "projectId" from jobs where id = t."jobId");
```

Comment:
will it cause the table to lockup?

Reply:
Do not use sub-queries. This can be done with a simple join.